### DISCUSSION TITLE
可以吧章节的测试也加进去吗？

### DISCUSSION THREAD

**Discussion by saye04** [aglorice/new_xxt#19] [Ideas]
大佬你好，请问可以把章节测试也加进去吗，因为目前xxt用的最多的其实还是那个章节视频和测验😭😭

**Comment by aglorice** [comment]
> 大佬你好，请问可以把章节测试也加进去吗，因为目前xxt用的最多的其实还是那个章节视频和测验😭😭

你可以用这个项目，因为有很多的项目都实现了这个功能，我就没弄了。https://github.com/SocialSisterYi/CxKitty

**Reply by saye04** [reply]
谢谢你，但是他的导出很多题似乎没有答案，只是题目。我感觉我不是很会用，想自己弄个题库文件的😂

**Reply by aglorice** [reply]
我暂时不会去弄这个功能

**Reply by saye04** [reply]
> 我暂时不会去弄这个功能

好的，我再去想想其他办法，谢谢你😀
